tests/integration/events/simple/with_update_test.go

```diff
@@ -66,14 +66,14 @@ func TestEventsSimpleWithUpdate(t *testing.T) {
 		ExpectedUpdates: []testUtils.ExpectedUpdate{
 			{
 				DocID: immutable.Some(docID1),
-				Cid:   immutable.Some("bafybeif757a4mdwimqwl24ujjnao6xlajiajz2hwuleopnptusuttri6zu"),
+				Cid:   immutable.Some("bafyreifvrmwmlwtglxe3afki36spu6d5qs6vvza57kxs4giyi53r5vbbnu"),
```

Comment:
question: I skimmed very quickly over the tests, at a glance they all seemed to be cid changes only.  Are there any public behaviour changes in this PR?

Reply:
There are no public behaviour changes :)

Reply:
Nice, cheers :)